package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vscode/js-debug-browsers",
-  "version": "1.0.8",
+  "version": "1.1.0",
```

Comment:
Just assumed that increasing the minor version was fine; I can also increase the patch version.